src/App.js

```diff
@@ -1,5 +1,5 @@
 /* eslint-disable react/jsx-props-no-spreading */
```

Comment:
Is this necessary btw?

Reply:
I will get the shared config so we can check then